### PR TITLE
Initial squashed import of internal repository

### PR DESCRIPTION
**Pull request by mattsheppard-instaclustr**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+pushd $(dirname "$0")
+docker run --rm -v $PWD:/home/marp/app/ -e LANG=$LANG marpteam/marp-cli --html docs/slides.md --pdf -o target/slides.pdf
+docker run --rm -v $PWD:/home/marp/app/ -e LANG=$LANG marpteam/marp-cli --html docs/slides.md --pptx -o target/slides.pptx
+docker run --rm -v $PWD:/home/marp/app/ -e LANG=$LANG marpteam/marp-cli --html docs/slides.md --html -o target/slides.html
+popd
```
Comment (parislarkins) [inline build.sh]: Minor one here, but it's flagged that there's no newline at the end of the file

**Review by parislarkins — CHANGES_REQUESTED** [review]
Looks good overall, just one minor change requested to add a newline to the end of `build.sh`

File: build.sh (hunk 1 above)
Comment (mattsheppard-instaclustr) [reply build.sh]: Thanks - Should be fixed now 😄

**Review by parislarkins — APPROVED** [review]
Looks good!

**Pull request merged**